src/UserInput/ButtonDuo.react.tsx

```diff
@@ -15,8 +15,8 @@ interface Props {
   secondaryBlocking?: boolean;
   secondaryLoading?: boolean;
   secondaryDisabled?: boolean;
-  secondaryLink?: boolean;
   style?: StyleProp<ViewStyle>;
+  rankOfSecondary?: "secondary" | "tertiary"
```

Comment:
what do you think about `secondaryRank`, to be consistent with the other `secondaryFoo` props?